tmt/steps/execute/internal.py

```diff
@@ -153,46 +187,6 @@ def __init__(self, **kwargs: Any):
         self._previous_progress_message = ""
         self.scripts = SCRIPTS
 
-    # TODO: consider switching to utils.updatable_message() - might need more
-    # work, since use of _show_progress is split over several methods.
-    def _show_progress(self, progress: str, test_name: str,
-                       finish: bool = False) -> None:
-        """
-        Show an interactive progress bar in non-verbose mode.
-
-        If the output is not an interactive terminal, or progress bar is
-        disabled using an option, just output the message as info without
-        utilising \r. If finish is True, overwrite the previous progress bar.
-        """
-        # Verbose mode outputs other information, using \r to
-        # create a status bar wouldn't work.
-        if self.verbosity_level:
-            return
-
-        # No progress if terminal not attached or explicitly disabled
-        if not sys.stdout.isatty() or self.data.no_progress_bar:
```

Comment:
The `isatty()` case is not handled any more?

Reply:
I believe it is still handled, but by the `UpdatableMessage` class, https://github.com/teemtee/tmt/blob/main/tmt/utils.py#L4837 and then `self.enabled` is checked by `_update_message_area()` method - if disabled, no output is emitted.

Reply:
Ah, I see, so it's covered there. Good!